src/partials/collection.html

```diff
@@ -1,3 +1,53 @@
 <section class="collection">
```

Comment:
1. Should be add id in section `<section class="collection" id="collection">`

2. After tag section  should be `<div class="collection-container container">`. In container should be added all your code.

Reply:
Done